PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py

```diff
@@ -55,3 +55,27 @@ def miniAOD_customizeMETFiltersFastSim(process):
 phase2_hgcal.toReplaceWith( Flag_HBHENoiseIsoFilter, cms.Path() )
 phase2_hgcal.toReplaceWith( Flag_eeBadScFilter, cms.Path() )
 
+metFilterPathsTask = cms.Task(
+    HBHENoiseFilterResultProducer,
+    HBHENoiseFilter,
```

Comment:
can this task be generated directly from ```allMetFilterPaths```?
tasks corresponding to individual paths defined above can be added and then automatically placed into the large metFilterPathsTask

I'm trying to reduce the number of distantly connected places where edits should be   made to get a new filter